packages/ui/src/memberships/types/memberships.ts

```diff
@@ -27,7 +30,21 @@ export interface Member {
   referredBy?: ID
 }
 
-export interface DetailedMember extends Member {
+export interface Membership extends MemberCommon {
+  type?: 'Membership'
+}
+
+export interface MMember extends MemberCommon {
+  type: 'Members'
+  isConcilMember: boolean
+  totalBalanced: number
+  totalStacked: number
+}
+
+export type Member = Membership | MMember
+export type Members = Membership[] | MMember[]
+
+export interface DetailedMember extends Membership {
```

Comment:
Initially, I wanted to push some changes to those types but after second thought I think that this change should also be reverted.

Sorry for that but the domain behind those fields wasn't properly described. You can assume that:

*   there will be a hook for getting member's balance and stake, you can use simply `0` for those values in the component for now
*   the same is with `isCouncil` I'm not 100% sure, but it looks like that part of the Joystream GraphQL queries was not yet designed. Having in mind that most members will not be council members please also use "no" here in the list.